crates/slightfile/tests/good/dup_good-1.toml

```diff
@@ -0,0 +1,12 @@
+specversion = "0.2"
```

Comment:
~change this file's name to match the other ones using hyphens rather than underscores.